skins/cat17/src/sass/layouts/pages/use_of_funds.scss

```diff
@@ -0,0 +1,59 @@
+.page-use-of-funds {
+	.inline {
+		display: flex;
```

Comment:
Generally I find `flex` easy to use for positioning elements. I used it in this particular case because I want the icon to be in the center of the `y` axis. But you're right that then I don't need `float: right`.